x/deployment/client/cli/tx.go

```diff
@@ -135,3 +136,50 @@ func cmdUpdate(key string, cdc *codec.Codec) *cobra.Command {
 	AddDeploymentIDFlags(cmd.Flags())
 	return cmd
 }
+
+// getTxGroupCmd returns the Group modification commands
+func getTxGroupCmd(key string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        "group",
```

Comment:
I think we should follow the pattern of the other ones here - `group-close` (one command).

It's 100% debatable whether or not it's the right pattern, but let's at least keep them consistent.

Reply:
I'm a little confused as to what `group-close` as one command is consistent with? Typically our CLI structure follows the pattern `<uttility group> <object...> <action verb>`. eg `akashctl tx deployment close` 

Or just because `group-close` is tightly coupled to a `deployment`?

Reply:
If by utility group you mean `tx`, `query`, etc... it's:

```
<utility-group> <module> <...>
```

for `query` commands things are mostly as you say but that's not the case right now for `tx` commands:

```
/akash ❯❯❯ ./akashctl help tx market                                                                                                                                                master ✭ ◼
Transaction subcommands

Usage:
  akashctl tx market [flags]
  akashctl tx market [command]

Available Commands:
  bid-close   Close a market bid
  bid-create  Create a market bid
  order-close Close a market order
```

Reply:
ahh okay, thanks. I'll drop the extra command group.